packages/manager/apps/pci-gateway/src/pages/add/LocationStep.tsx

```diff
@@ -98,21 +99,25 @@ export const LocationStep = () => {
     }
   }, [searchParams, state.regions]);
 
+  const has3AZ = useMemo(() => isRegionWith3AZ(state.regions), [state.regions]);
+
+  const pciCommonProperties = usePCICommonContextFactory({ has3AZ });
```

Comment:
Naming should be agnostic from the library :)

Reply:
Since it is the pci common context, how would you name it ?

Reply:
Could be directly `const meta = usePCICommonContextFactory({ has3AZ });` or something like that ;)